templates/vhost/_ssl.erb

```diff
@@ -16,8 +16,6 @@
   <%- if @ssl_honorcipherorder -%>
   SSLHonorCipherOrder     <%= @ssl_honorcipherorder %>
   <%- end -%>
-  <%- if @ssl_verify_client -%>
```

Comment:
This `if` cases out everything between lines 19 and 39. Even if you want to move the `SSLVerifyClient` directive lower, you still need to keep the `if` here.